### PR TITLE
Add a notebook demonstrating using the entity service with blocking

### PR DESCRIPTION
**Pull request by hardbyte**

### CODE REVIEW

File: docs/tutorial/tutorial_blocking.ipynb [hunk 1]
```diff
@@ -620,18 +634,18 @@
...
+    },
+    "source": []
    }
   }
  },
  "nbformat": 4,
  "nbformat_minor": 4
-}
\ No newline at end of file
+}
```
Comment (JoyceXinyueWang) [inline docs/tutorial/tutorial_blocking.ipynb]: Hi Brian, are you sure you have committed the right changes? All changes there seems to be irrelevant cell output change.  I didn't see anything relate to entity service here

**Comment by hardbyte** [comment]
Oh sorry I had both blocklib and anonlink-client open in PyCharm at the same time and it made a branch in both projects. Sorry just ignore this!
